### ISSUE TITLE
[Feature Request] Sub-Identities

### ISSUE THREAD

**Issue by BeholdersEye** [oxen-io/session-desktop#2519]
Just as cryptocurrency wallets generate sub-addresses, it would be useful if Session allowed users to deterministically derive sub-identities from their main identity.

These sub-identities could be treated as disposable in the same way that cryptocurrency subaddresses are used only once.

Sub-identities would have their own profile information.

Messages sent to a sub-identity would be displayed alongside messages sent to the main identity but when replying there would be a notice above the text input like "Replying as: <sub-identity>"

**Comment by KeeJef** [comment]
Closing this as a duplicate of #1809 which would essentially be the same thing

**Comment by BeholdersEye** [comment]
The described implementation details make this feature request distinct from the suggest you linked. Request reopening.
